api/v1alpha1/meshsync_types_test.go

```diff
@@ -110,7 +110,9 @@ var _ = Describe("The test case for the meshsync CRDs", func() {
 				},
 			}
 			By("Update the status of the meshsync CRDs")
-			err := fakeClient.Status().Update(context, meshSync, &client.UpdateOptions{FieldManager: FileManager})
+			err := fakeClient.Status().Update(context, meshSync, &client.SubResourceUpdateOptions{
```

Comment:
Hi, @acald-creator thanks for your contribution. Please check the CI issue first. I consider the test case "The meshsync CRDs update the status of the resources" is useful, I mean do we need to check the status?

Reply:
@Aisuko I saw the CI issue, I will check to see if I can get this resolved.